src/averageAges.js

```diff
@@ -20,6 +20,38 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+
+  // ----------------- version 1
+  /*
+  const personFromNeededCentury = century
+    ? people.filter(person => Math.ceil(people.died / 100) === century)
+    : people;
+
+  const maleOnly=personFromNeededCentury.filter(person=>person.sex === 'm');
+
+  const totalAge = maleOnly.reduce(
+    (prev, person) => prev + (person.died - person.born), 0
+  );
+
+  const averageAge = totalAge / maleOnly.length;
+
+  return averageAge;
+  */
+
+  // ----------------- version 2
+
+  const men = people.filter(person => person.sex === 'm')
```

Comment:
Consider using constants instead of magic strings

Reply:
Thanks. I've added a function to get men from the list.